packages/@react-spectrum/buttongroup/src/ButtonGroup.tsx

```diff
@@ -61,6 +61,8 @@ function ButtonGroup(props: SpectrumButtonGroupProps, ref: DOMRef<HTMLDivElement
     if (!dirty) {
       setDirty(true);
     }
+  // Don't add dirty to dep array since it will cause infinite loop
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why/how did this end up in here?

Reply:
Noticed it when doing a lint locally, and remembered a suggestion (forget from who) to disabled the lint warning so calling `--fix` wouldn't break everything by putting `dirty` into the dep array 

Coulda sworn we had added this before, but dunno why it is here again

Reply:
i thought we had too, haha, well, this is fine